js/main.js

```diff
@@ -275,8 +276,10 @@ $(function() {
             }
         };
         var used_errcodes = {};
-        var rda = $('#result_display_area');
-        rda.html('');
+        if (typeof addWarningCustom !== 'function') {
```

Comment:
any sense to write this at line 14?

```
if (addWarningCustom && typeof addWarningCustom !== 'function') { throw new Error('addWarningCustom must be a function(...)'); }
```

and check for undefinedness only afterwards?